src/modules/TodoList.js

```diff
@@ -0,0 +1,34 @@
+import Todo from './Todo.js';
+
+export default class TodoList {
+  constructor(todolist) {
+    this.todolist = todolist;
+  }
+
+  addTask(description, index) {
+    const newTask = new Todo(description, index);
+    this.todolist.push(newTask);
+  }
+
+  removeTask(index) {
+    this.todolist.splice((index - 1), 1);
+    const updatedTodoList = this.todolist.map((object) => {
+      if (object.index > index) {
+        const ind = object.index - 1;
+        return { ...object, index: ind };
+      }
+      return object;
+    });
+    this.todolist = updatedTodoList;
+  }
```

Comment:
- [ ] Good job implementing the remove functionality but at the moment, it does not work quite and does not update the screen immediately after an item is deleted.

Reply:
Hi The issue has been resolved.

Thanks